src/WebJobs.Script.Scaling/WebJobs.Script.Scaling.csproj

```diff
@@ -0,0 +1,100 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{292A9038-5807-4095-B37F-9A77170D7402}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>Microsoft.Azure.WebJobs.Script.Scaling</RootNamespace>
+    <AssemblyName>Microsoft.Azure.WebJobs.Script.Scaling</AssemblyName>
```

Comment:
Is this library specific to Functions? I thought you said it was a general library. That would influence the name we choose for it. The naming you chose here for the project and assembly are correct if this is only for Functions though.